tests/v2/actions/provisioning/verify.go

```diff
@@ -53,12 +53,15 @@ const (
 
 // VerifyRKE1Cluster validates that the RKE1 cluster and its resources are in a good state, matching a given config.
 func VerifyRKE1Cluster(t *testing.T, client *rancher.Client, clustersConfig *clusters.ClusterConfig, cluster *management.Cluster) {
+	logrus.Infof("Try 1: Cluster is %v", cluster)
```

Comment:
note: this is just for debugging via Jenkins runs. The tests passes on my local machine, but fails on Jenkins runs - likely due to timing issues.